install/kubernetes/helm/Istio/charts/initializer/templates/istio-initializer.yaml

```diff
@@ -0,0 +1,86 @@
+{{- $istio_namespace := .Values.global.namespace }}
+{{- $pilot_hub := .Values.global.pilot_hub }}
+{{- $pilot_tag := .Values.global.pilot_tag }}
+################################
+# Istio initializer
+################################
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: istio-inject
+  namespace: {{ $istio_namespace }}
+data:
+  config: |-
+    policy: "enabled"
+    namespaces: [""] # everything, aka v1.NamepsaceAll, aka cluster-wide
+    initializerName: "sidecar.initializer.istio.io"
+    params:
+      initImage: {{$pilot_hub}}/proxy_init:{{$pilot_tag}}
+      proxyImage: {{$pilot_hub}}/proxy_debug:{{$pilot_tag}}
+      verbosity: 2
+      version: {{$pilot_tag}}
+      meshConfigMapName: istio
+      imagePullPolicy: IfNotPresent
+      debugMode: true
```

Comment:
You can losing `IncludeIPRanges: "{{.Params.IncludeIPRanges}}"` which is very important to enable istio can talk to external services.